test/test_bot.py

```diff
@@ -172,6 +198,20 @@ def test_wrapper_reply(mockbot, triggerfactory):
     )
 
 
+def test_wrapper_reply_statusmsg(mockbot, triggerfactory):
```

Comment:
No `test_wrapper_say_statusmsg`? What about `bot.notice()` or `bot.action()`?

Reply:
Hu... you're right. I thought I had them covered.